README.md

```diff
@@ -169,64 +169,70 @@ couchbackup --db animaldb | openssl aes-128-cbc -pass pass:12345 > encrypted_ani
 openssl aes-128-cbc -d -in encrypted_animal.db -pass pass:12345 | couchrestore --db animaldb2
 ```
 
-Note that the content is unencrypted while it is being processed by the
-backup tool before it is piped to the encryption utility.
+Note that the content is not encrypted in the
+backup tool before piping to the encryption utility.
 
 ## What's in a backup file?
 
-A backup file is a text file where each line contains a JSON encoded array of up to `buffer-size` objects e.g.
+A backup file is a text file where each line is either a JSON object of backup metadata
+or a JSON array of backed up document revision objects, for example:
 
-```js
-    [{"a":1},{"a":2}...]
-    [{"a":501},{"a":502}...]
+```json
+{"name":"@cloudant/couchbackup","version":"2.9.10","mode":"full"}
+[{"_id": "1","a":1},{"_id": "2","a":2},...]
+[{"_id": "501","a":501},{"_id": "502","a":502}]
 ```
 
+The number of document revisions in a backup array varies. It typically has
+`buffer_size` elements, but may be more if there are also leaf revisions returned
+from the server or fewer if it is the last batch.
+
 ## What's in a log file?
 
-A log file contains a line:
+A log file has a line:
 
-- for every batch of document ids that need to be fetched e.g. `:t batch56 [{"id":"a"},{"id":"b"}]`
-- for every batch that has been fetched and stored e.g. `:d batch56`
-- to indicate that the changes feed was fully consumed e.g. `:changes_complete`
+- for every batch of document ids that couchbackup needs to fetch, for example: `:t batch56 [{"id":"a"},{"id":"b"}]`
+- for every batch that couchbackup has fetched and stored, for example: `:d batch56`
+- to indicate that the changes feed was fully consumed, for example: `:changes_complete`
 
-## What is shallow mode?
+## What's shallow mode?
 
-When you run `couchbackup` with `--mode shallow` a simpler backup is performed, only backing up the winning revisions
-of the database. No revision tokens are saved and any conflicting revisions are ignored. This is a faster, but less
-complete backup. Shallow backups cannot be resumed because they do not produce a log file.
+When you run `couchbackup` with `--mode shallow` couchbackup performs a simpler backup.
+It only backs up the winning revisions and ignores any conflicting revisions.
+This is a faster, but less complete backup. The `--log` and `---resume` options
+ are not valid for shallow mode backups.
 
-NOTE: Parallellism will not be in effect if `--mode shallow` is defined.
+NOTE: Parallelism does not apply for `--mode shallow`.
```

Comment:
```suggestion
**Note:** Parallelism does not apply for `--mode shallow`.
```

Reply:
If you think the comment above is worthwhile, we could probably put this all on one line:
```suggestion
**Note:** The `--log`,  `---resume`, and `--parallelism` options are not valid for shallow mode backups.
```